packages/react-native/Libraries/Text/Text/RCTTextView.h

```diff
@@ -23,6 +24,11 @@ NS_ASSUME_NONNULL_BEGIN
        descendantViews:(NSArray<RCTPlatformView *> *)descendantViews; // [macOS]
 
 #if TARGET_OS_OSX // [macOS
```

Comment:
I see you're putting it next to my function that I #def'd to OSX, but that was due to using NSRange which isn't cross-plat for iOS (in hindsight should've just used a pair of indexes instead of a platform type)

Reply:
The main argument to be made here is to have two separate `#if TARGET_OS_OSX` blocks for two unrelated items, which makes sense to me, so we can do that.

On an unrelated note, `NSRange` should be iOS-compatible, although I don't know off the top of my head if the implementation of `getRectForCharRange:` would be different on iOS and macOS. But that's obviously out of scope here.